src/inputs/prompt/prompt.jsx

```diff
@@ -155,30 +162,49 @@ class Prompt extends React.Component {
     }
 
     componentDidUpdate(prevProps) {
-        const { show: nextShowProp } = this.props;
-        const { show: prevShowProp } = prevProps;
+        const {
+            show: showProp,
+        } = this.props;
+
+        const {
+            show: prevShowProp,
+        } = prevProps;
+
+        const {
+            show: showState,
+        } = this.state;
 
-        if (nextShowProp !== prevShowProp) {
-            this.setState({ show: nextShowProp });
+        const {
+            show: prevShowState,
+        } = prevProps;
+
+        if (showProp !== prevShowProp) {
+            this.setState({
+                show: showProp,
+            }, () => {
+                this.focusNoButton();
+            });
+        }
+
+        if (showState && !prevShowState) {
+            this.focusNoButton();
         }
     }
 
     onClick(option) {
         const { onClick } = this.props;
         const { show } = this.state;
 
-        if (show) { return false; }
+        if (show) { return; }
 
         if (!isUndefined(onClick)) {
             onClick(option);
         } else {
             this.setState({ show: true });
         }
-
-        return false;
```

Comment:
We're sure `return false;` is unnecessary here?  It wasn't like serving to stop the event from propagating or anything like that, right?  I trust you ... just triple-checking ;-)

Reply:
It was to appease eslint because the a few lines above, we were returning `false`. I don't see why we'd need to return anything other than to get out of there if `show` is `true`

Reply:
Sounds good.  Resolving.